backend/src/controllers/user.controller.ts

```diff
@@ -60,6 +68,17 @@ export const update = async (req: Request, res: Response): Promise<Response> =>
   }
   const update = req.body as Partial<IUser>;
   try {
+    if (update.email) {
```

Comment:
Lets be consecutive in naming convention. Please change `update` into `body` as in previous example